pysamp/actor.py

```diff
@@ -141,10 +141,30 @@ def is_valid(self) -> bool:
 
     @event("OnActorStreamIn")
     def on_stream_in(cls, actorid: int, forplayerid: int):
+        """This callback is called when an actor is streamed in\
+        by a player's client.
+
+        :param int actorid: The ID of the actor.
+        :param int forplayerid: The ID of the player\
+        that streamed the actor in.
+        :returns: No return value.
+
+        .. note:: This callbackcan also be called by NPC.
```

Comment:
There are some simple space-errors here and there. Could you go through these before we add?